.github/workflows/test.yaml

```diff
@@ -44,7 +44,9 @@ jobs:
       tc-keras: ${{ steps.filter.outputs.tc-keras }}
       tc-pytorch: ${{ steps.filter.outputs.tc-pytorch }}
       tc-tensorflow: ${{ steps.filter.outputs.tc-tensorflow }}
-      ic-pytorch: ${{ steps.filter.outputs.ic-pytorch }}
```

Comment:
why is this deleted?

Reply:
@anthonycorletti we only had 1 notebook being tested ic-pytorch, but we have 3 IC notebooks, all pytorch. So I just added the missing 2 and renamed appropriately